domain-models-interface-extensions/src/main/java/org/folio/rest/tools/ClientGenerator.java

```diff
@@ -383,17 +388,17 @@ public void generateMethodMeta(String methodName, JsonObject params, String url,
     ifClause2._then().directStatement("request.putHeader(\"X-Okapi-Url\", okapiUrl);");
 
     /* add response handler to each function */
-    JClass handler = jcodeModel.ref(Handler.class).narrow(HttpClientResponse.class);
+    JClass handler = jcodeModel.ref(Handler.class).narrow(jcodeModel.ref(AsyncResult.class).narrow(jcodeModel.ref(HttpResponse.class).narrow(Buffer.class)));
     jmCreate.param(handler, "responseHandler");
 
     /* if we need to pass data in the body */
     if(bodyContentExists[0]){
       body.directStatement("request.putHeader(\"Content-Length\", buffer.length()+\"\");");
-      body.directStatement("request.setChunked(true);");
-      body.directStatement("request.write(buffer);");
+      //body.directStatement("request.setChunked(true);"); //TODO: consider adding this somehow
```

Comment:
Remove the lines with content-length and setChunked.
Then WebClient may chunk the request: https://vertx-web-site.github.io/docs/vertx-web-client/java/#_writing_request_bodies